craco.config.js

```diff
@@ -34,4 +34,14 @@ module.exports = {
     eslint: {
         mode: ESLINT_MODES.file
     },
+    jest: {
```

Comment:
it would be better if you make a different file for config. check croco configurations